bin/branding.list

```diff
@@ -10,6 +10,7 @@ AUTHOR
 LICENSE
 HOMEPAGE
 CHANGELOG_PAGE
+ORGANIZATION
```

Comment:
disagree - if you are publishing and its empty things get messed up!  

Lots of things are required but only for publishing (and its added to the test files that ship - so it will be set by default if you use the test branding included in this project.  (e.g RPM_URL is unused until publishing)